packages/viewer/src/lib/interactions/interactionApi.ts

```diff
@@ -372,26 +372,39 @@ export abstract class InteractionApi<T extends Camera = Camera> {
   public async rotateCamera(delta: Point.Point): Promise<void> {
     return this.transformCamera(({ camera, viewport }) => {
       const upVector = Vector3.normalize(camera.up);
-      const lookAt = Vector3.normalize(
+      const viewVector = Vector3.normalize(
         Vector3.subtract(camera.lookAt, camera.position)
       );
 
-      const crossX = Vector3.cross(upVector, lookAt);
-      const crossY = Vector3.cross(lookAt, crossX);
+      const crossX = Vector3.cross(upVector, viewVector);
+      const crossY = Vector3.cross(viewVector, crossX);
 
       const mouseToWorld = Vector3.normalize({
         x: delta.x * crossX.x + delta.y * crossY.x,
         y: delta.x * crossX.y + delta.y * crossY.y,
         z: delta.x * crossX.z + delta.y * crossY.z,
       });
 
-      const rotationAxis = Vector3.cross(mouseToWorld, lookAt);
+      const rotationAxis = Vector3.cross(mouseToWorld, viewVector);
 
       const epsilonX = (3.0 * Math.PI * delta.x) / viewport.width;
       const epsilonY = (3.0 * Math.PI * delta.y) / viewport.height;
       const angle = Math.abs(epsilonX) + Math.abs(epsilonY);
 
-      return camera.rotateAroundAxis(angle, rotationAxis);
+      const updated = camera.rotateAroundAxis(angle, rotationAxis);
+
+      return updated.update({
+        // Scale the lookAt point to the same length as the distance to the
+        // center of the bounding box to maintain zoom and pan behavior.
+        lookAt: Vector3.add(
+          Vector3.scale(
+            Math.abs(camera.signedDistanceToBoundingBoxCenter()) /
+              Vector3.magnitude(updated.viewVector),
+            updated.viewVector
+          ),
+          updated.position
+        ),
+      });
```

Comment:
Unless these changes are to fix a camera positioning issue, I wouldn't expect any changes here to fix computation of the depth buffer.